### PR TITLE
Get right Android SDK Path on Linux, Mac and Windows

### PR DESCRIPTION
**Pull request by JcMinarro**
Use AndroidSdkUtils to obtain Android SDK Path instead to ask to system about its environment variables

### CODE REVIEW

File: src/main/java/com/github/pedrovgs/androidwifiadb/AndroidWiFiADB.java [hunk 1]
```diff
@@ -24,7 +26,7 @@
   private final ADB adb;
   private final View view;
 
-  public AndroidWiFiADB(ADB adb, View view) {
+    public AndroidWiFiADB(ADB adb, View view) {
```
Comment (pedrovgs) [inline src/main/java/com/github/pedrovgs/androidwifiadb/AndroidWiFiADB.java]: Can you remove this space?

File: src/main/java/com/github/pedrovgs/androidwifiadb/AndroidWiFiADB.java [hunk 2]
```diff
@@ -53,4 +55,8 @@ public void connectDevices() {
   private boolean isADBInstalled() {
     return adb.isInstalled();
   }
+
+    public void updateProject(Project project) {
+        this.adb.updateProject(project);
+    }
```
Comment (pedrovgs) [inline src/main/java/com/github/pedrovgs/androidwifiadb/AndroidWiFiADB.java]: Can you review the indentation here?

File: src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java [hunk 3]
```diff
@@ -78,19 +76,21 @@ private boolean connectDevice(String deviceIp) {
     return connectOutput.contains("connected");
   }
 
-  private String getAdbPath() {
-    String androidSdkPath = EnvironmentUtil.getValue(ANDROID_ENV_VAR_NAME);
-    if (androidSdkPath == null || androidSdkPath.isEmpty()) {
-      return "";
+
+  private String getAdbPath(Project project) {
```
Comment (pedrovgs) [inline src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java]: If I'm not wrong, the parameter `Project project` is not needed.

File: src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -21,7 +21,7 @@
   <idea-version since-build="131"/>
 
   <extensions defaultExtensionNs="com.intellij">
-
+      <defaultProjectTypeProvider type="Android"/>
```
Comment (pedrovgs) [inline src/main/resources/META-INF/plugin.xml]: Can you review the indentation here?

**Comment by pedrovgs** [comment]
If I'm not wrong, with this awesome PR to declare the `ANDROID_HOME` environment variable is not needed. Can you review the `README.md` file and remove the line where we says this environment variable is mandatory.

**Comment by pedrovgs** [comment]
@JcMinarro I don't have more comments related to the code, it seems ok :). Please review my comments and let me know once the PR be updated to check if your implementation is working as should.

File: src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java (hunk 3 above)
Comment (JcMinarro) [inline src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java]: You are right ;)

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ Android WiFi ADB - IntelliJ/Android Studio Plugin [![Build Status](https://travi
 
 **Android WiFI ADB plugin adds a button ![Android WiFi ADB Button][5] to your IntelliJ/Android Studio Toolbar** to connect your device to your computer over WiFi.  
 
-*To use this plugin you have to declare the ``ANDROID_HOME`` env var properly.*
+*This plugin only works when you are working with an Android Project configured on your IntelliJ/Android Studio.*
```
Comment (pedrovgs) [inline README.md]: This sentence should be: `To use this plugin the project opened in your IntelliJ/Android Studio has to be an Android project configured with the Android SDK`

**Comment by pedrovgs** [comment]
@JcMinarro something is wrong with this PR. If I execute the plugin from the IDE I get an assertion error trying to execute the line:

``` java
return AndroidSdkUtils.isAndroidSdkAvailable();
```

The `AndroidSdkUtils` has an assert failing. This is the message it throws: `Method threw 'java.lang.AssertionError' exception.`.

If I build the plugin using `./gradlew buildPlugin` I can't install the plugin in Android Studio. The error message says: `Error: Plugin 'Android WiFi ADB is incompatible with this installation'.`

I'd like to merge this PR to avoid the usage of `ANDROID_HOME` as an environment variable, but I don't know why it's not working. Can you take a look?

**Comment by JcMinarro** [comment]
The plugin is not compatible with AS because on "autogenerate plugin.xml file" is override the "since-build version" of the idea. It is overrided by IntelliJ Gradle Plugin. I have done some changes.
Be sure that you have setup Android-Support properly on your IntelliJ ;)

**Comment by pedrovgs** [comment]
Awesome PR @JcMinarro I'm going to merge it and generate a new version. AndroidWiFiADB 1.2. Thank you so much.

**Pull request merged**